### PR TITLE
Add gasp to TTF if info.openTypeGaspRangeRecords is present

### PR DESCRIPTION
**Pull request by moyogo**

### CODE REVIEW

File: Lib/ufo2ft/outlineOTF.py [hunk 1]
```diff
@@ -942,6 +953,8 @@ def setupTable_post(self):
 
     def setupOtherTables(self):
         self.setupTable_glyf()
+        if getattr(self.ufo.info, "openTypeGaspRangeRecords"):
```
Comment (adrientetar) [inline Lib/ufo2ft/outlineOTF.py]: Is the getattr for robofab compability? With defcon I guess you can always call self.ufo.info.openTypeGaspRangeRecords

Comment (moyogo) [inline Lib/ufo2ft/outlineOTF.py]: Good point. Fixed.

**Comment by anthrotype** [comment]
mind adding small unit test? 😁

**Comment by anthrotype** [comment]
excellent, thanks!

I think we should move the test modules outside of the package and not install them, especially if we are going to add more test UFOs in the future.

We can do that in a separate PR.

**Pull request merged**